src/IrcFeatures/DccChat.cs

```diff
@@ -130,8 +130,8 @@ internal override void InitWork(Object stateInfo)
 
             DccChatStartEvent(new DccEventArgs(this));
 
-            _sr = new StreamReader(Connection.GetStream(), Irc.Encoding);
-            _sw = new StreamWriter(Connection.GetStream(), Irc.Encoding);
+            _sr = new StreamReader(Connection.GetStream(), System.Text.Encoding.Default);
+            _sw = new StreamWriter(Connection.GetStream(), System.Text.Encoding.Default);
```

Comment:
https://github.com/meebey/SmartIrc4net/blob/master/src/IrcFeatures/DccChat.cs#L63 shows how it worked before. The ctor of DccChat() takes IrcFeatures and that class extends IrcConnection which had Encoding as a property before. Thus I suggested to add Encoding to the Transport interface so it can use Irc.Transport.Encoding here.